tests/components/litterrobot/conftest.py

```diff
@@ -33,3 +37,19 @@ def mock_hub(hass):
     hub.coordinator.last_update_success = True
     hub.account.robots = [create_mock_robot(hass)]
     return hub
+
+
+async def setup_hub(hass, mock_hub, platform_domain):
+    """Load a Litter-Robot platform with the provided hub."""
+    entry = MockConfigEntry(
+        domain=litterrobot.DOMAIN,
+        data=CONFIG[litterrobot.DOMAIN],
+    )
+    entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.litterrobot.LitterRobotHub",
+        return_value=mock_hub,
+    ):
+        await hass.config_entries.async_forward_entry_setup(entry, platform_domain)
```

Comment:
We can patch the constant list of platforms instead to limit which platform is set up. We should set up the integration with `hass.config_entries.async_setup`.